SwiftStarter/SwiftStarter/Supporting Files/sampleHTMLCode.html

```diff
@@ -8,15 +8,48 @@
         <h2><center>CleverTap Webview</center></h2><br></br>
         <button onclick="myFunction()">Record Event</button>
         <p id="demo"> </p>
-        <script> function myFunction() {
-            document.getElementById("demo").innerHTML = 'Hey, whats up?';
-            var props = {foo: 'xyz', lang: 'French'};
-            var message = { action:'recordEventWithProps', event:'CT Webview event', properties: props};
-            
-            if(window.webkit
-                     && window.webkit.messageHandlers
-                     && window.webkit.messageHandlers.clevertap){
+        <button onclick="recordChargedEvent()">Record Charged Event</button>
```

Comment:
Hey @akashvercetti, Let's add examples for `profilePush`, `profileSetMultiValues`, `profileAddMultiValues` and `profileRemoveMultiValues`.

Reply:
Sure